testdata/simple/5a75f537726934e4b62833ab6d5dca41/tests/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "_id": "62dba0159041307f697e6ccc",
+    "build_id": "5a75f537726934e4b62833ab6d5dca41",
+    "build_name": "MCI_enterprise-rhel_job0",
+    "name": "geo_max:CheckReplOplogs",
+    "command": "",
+    "started": "2022-07-23T07:15:33.967Z",
+    "ended": null,
+    "info": {
+      "task_id": "mongodb_mongo_master_enterprise_rhel_80_64_bit_multiversion_all_feature_flags_retryable_writes_downgrade_last_continuous_2_enterprise_f98b3361fbab4e02683325cc0e6ebaa69d6af1df_22_07_22_11_24_37"
+    },
+    "phase": "retryable_writes_downgrade_last_continuous_2-enterprise_0",
+    "seq": 1
```

Comment:
this field seems specific to the db based implementation where we write new chunks to the db starting from the test's sequence number.

Reply:
Removed